concrete/src/Url/Resolver/CanonicalUrlResolver.php

```diff
@@ -32,18 +39,9 @@ public function __construct(Application $app, Request $request)
     }
 
     /**
-     * Resolve url's from any type of input.
-     *
-     * This method MUST either return a `\League\URL\URL` when a url is resolved
-     * or null when a url cannot be resolved.
-     *
-     * If the first argument provided is a page object, we will use that object to determine the site tree
-     * (and thus the canonical url) to use.
-     *
-     * @param array $arguments A list of the arguments
-     * @param \League\URL\URLInterface $resolved
+     * {@inheritdoc}
      *
-     * @return \League\URL\URLInterface
+     * @see \Concrete\Core\Url\Resolver\UrlResolverInterface::resolve()
```

Comment:
I don't think we need the inheritdoc and the see

Reply:
I love explicitly saying where the doc is defined (it's very useful for classes that extend other classes and/or implement multiple interfaces)